#! /bin/sh /usr/share/dpatch/dpatch-run
## autconf_order.dpatch by Tzafrir Cohen <tzafrir.cohen@xorcom.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix autoconf orderng. 
## From http://svn.digium.com/view/asterisk?view=rev&rev=48960 . Should
## part of 1.4.1 .

@DPATCH@
--- asterisk-1.4.0/configure.ac   2006/12/25 06:42:15     48959
+++ asterisk-1.4.0/configure.ac   2006/12/25 17:04:48     48960
@@ -21,6 +21,9 @@
 AC_COPYRIGHT("Asterisk")
 AC_REVISION($Revision$)
 
+AC_GNU_SOURCE
+AC_USE_SYSTEM_EXTENSIONS
+
 case "${host_os}" in
      freebsd*)
      ac_default_prefix=/usr/local
@@ -89,9 +92,6 @@
 fi
 AC_SUBST(PBX_OSREV)
 
-# This needs to be before any macros that use the C compiler
-AC_GNU_SOURCE
-
 AH_TOP(
 #ifndef ASTERISK_AUTOCONFIG_H
 #define ASTERISK_AUTOCONFIG_H

